frontend/main.js

```diff
@@ -90,7 +90,7 @@ async function startApp () {
       return {
         ephemeral: {
           syncs: [],
-          finishedLogin: 'no'
+          finishedLogin: !!this.$store.state.loggedIn
```

Comment:
This is definitely going to break things since it modifies a string to being a boolean but doesn't modify any of the other code to take that into account.

Reply:
updated to return 'yes' or 'no'